resources/carrier/en/374.txt

```diff
@@ -15,15 +15,15 @@
 # Data from Wikipedia:
 # http://en.wikipedia.org/wiki/Telephone_numbers_in_Armenia#Mobile_network_codes
 
-37441|Orange
+37441|Ucom
```

Comment:
Please provide evidence that this change took place.
To contribute evidence please see our [contribution guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md#provide-supporting-evidence).